### PR TITLE
feat(CKB): add sunburst & gauge chart spec

### PR DESCRIPTION
**Pull request by cxxxxxn**
### PR includes

- [x] add sunburst & gauge chart spec and Chinese translation.
- [x] delete polar treemap (It's same with sunburst)
- [x] update treemap spec (it displays hierarchy data, so I think its category is not Statistic)
- [x] update pie chart, donut chart, rose chart spec

### CODE REVIEW

File: packages/knowledge/src/base.ts [hunk 1]
```diff
@@ -450,7 +450,7 @@ export const base: ChartKnowledgeBase = {
       { minQty: 1, maxQty: 1, fieldConditions: ['Nominal', 'Ordinal'] },
       { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] },
     ],
-    channel: ['Angle', 'Area', 'Color'],
+    channel: ['Angle', 'Color'],
```
Comment (neoddish) [inline packages/knowledge/src/base.ts:453]: I would say Area is one of the channels of pie charts. There are papers discussed this, I believe.

File: packages/knowledge/src/base.ts [hunk 2]
```diff
@@ -469,7 +469,7 @@ export const base: ChartKnowledgeBase = {
       { minQty: 1, maxQty: 1, fieldConditions: ['Nominal', 'Ordinal'] },
       { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] },
     ],
-    channel: ['ArcLength'],
+    channel: ['Angle', 'Color'],
```
Comment (neoddish) [inline packages/knowledge/src/base.ts:472]: reason?

File: packages/knowledge/src/base.ts [hunk 3]
```diff
@@ -501,7 +501,7 @@ export const base: ChartKnowledgeBase = {
     shape: ['Round'],
     dataPres: [
       { minQty: 1, maxQty: 1, fieldConditions: ['Nominal', 'Ordinal'] },
-      { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] },
+      { minQty: 1, maxQty: 2, fieldConditions: ['Interval'] },
```
Comment (neoddish) [inline packages/knowledge/src/base.ts:504]: why？

I suppose nominal or ordinal could be 2 (one for color channel), but 2 interval fields?

File: packages/knowledge/src/base.ts [hunk 4]
```diff
@@ -554,7 +554,7 @@ export const base: ChartKnowledgeBase = {
       'A visual representation of a data tree with nodes. Each node is displayed as a rectangle, sized and colored according to values that you assign.',
     purpose: ['Composition', 'Comparison', 'Hierarchy'],
     coord: ['Cartesian2D'],
-    category: ['Statistic'],
+    category: ['Graph'],
```
Comment (neoddish) [inline packages/knowledge/src/base.ts:557]: how could we define the boundary between statistic and graph?

File: packages/knowledge/src/base.ts [hunk 5]
```diff
@@ -1251,33 +1254,21 @@ export const base: ChartKnowledgeBase = {
     recRate: 'Recommended',
   },
 
-  polar_treemap: {
```
Comment (neoddish) [inline packages/knowledge/src/base.ts:1254]: what was this.....polar treemap....

File: packages/knowledge/src/chartID.ts [hunk 6]
```diff
@@ -77,7 +77,6 @@ export const CHART_ID_OPTIONS = [
   'bubble_map',
   'bubble_light_map',
   'packed_circles',
-  'polar_treemap',
```
Comment (neoddish) [inline packages/knowledge/src/chartID.ts:80]: did u run tests or lerna boot? this could make one test case fail

**Review by neoddish — CHANGES_REQUESTED** [review]

File: packages/knowledge/src/base.ts (hunk 3 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:504]: I think one interval field for angle and one interval field for length.

File: packages/knowledge/src/base.ts (hunk 2 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:472]: I think the donut chart is almost identical to the pie except that it is hollow in the middle.
For example, in [echarts](https://echarts.apache.org/examples/zh/editor.html?c=pie-borderRadius), [vaga-lite](https://vega.github.io/vega-lite/examples/arc_donut.html), they both display the donut chart by setting the innerRadius (more like a style change).
In [G2Plot](https://g2plot.antv.vision/en/examples/case/statistical-scenario#pie), it uses angle and color field also.

File: packages/knowledge/src/base.ts (hunk 3 above)
Comment (Esorakouki) [reply packages/knowledge/src/base.ts:504]: length?  you mean 'width'?   I think the width of  segments is a constant，or determined by its hierarchical relationship.

File: packages/knowledge/src/chartID.ts (hunk 6 above)
Comment (cxxxxxn) [reply packages/knowledge/src/chartID.ts:80]: I didn't. 
I've solved it now.

File: packages/knowledge/src/base.ts (hunk 3 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:504]: Oh, I confused it with the radial chart. So 2 nominal or ordinal fields, one for angle and one for color?

Comment (Esorakouki) [reply packages/knowledge/src/base.ts:504]: 1 interval field for angle channel, other fields (one or more than one, nominal or ordinals ) build a hierarchical relationship(color channel).

Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:504]: Can the rose chart show a hierarchical relationship?

Comment (Esorakouki) [reply packages/knowledge/src/base.ts:504]: Rose chart may not, its sectors  are equal angles.

File: packages/knowledge/src/base.ts (hunk 1 above)
Comment (neoddish) [reply packages/knowledge/src/base.ts:453]: users will see them in diff ways.

e.g. pie charts have 'angles' while donut charts have not, but both of them have 'area'. Its the way that an average user can see.

File: packages/knowledge/src/base.ts (hunk 2 above)
Comment (neoddish) [reply packages/knowledge/src/base.ts:472]: OIC, let me try to explain this... we set these props in CKB not by how the grammar of graphics defines the cases but how average users will understand the cases.

e.g. in a donut chart, users can't get the angle, actually. They may see the proportion via area, however, area is not accurate in donuts. So, when the inner and outer radius are getting close, in fact, it the length of arcs by which users get the difference.

File: packages/knowledge/src/base.ts (hunk 3 above)
Comment (neoddish) [reply packages/knowledge/src/base.ts:504]: so, for rose chart shall we set here: 
```
dataPres: [
  { minQty: 1, maxQty: 2, fieldConditions: ['Nominal', 'Ordinal'] }, 
  { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] }
],
```

File: packages/knowledge/src/base.ts (hunk 5 above)
Comment (neoddish) [reply packages/knowledge/src/base.ts:1254]: got it.....

https://g2.antv.vision/en/examples/relation/relation#polar-treemap

Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:1254]: I found the definition of a polar treemap on this site https://resources.wolframcloud.com/FunctionRepository/resources/PolarTreemapPlot/.

It looks the same as sunbrust, so I deleted the polar treemap.

File: packages/knowledge/src/base.ts (hunk 2 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:472]: so, it should be [ 'ArcLength', 'Color' ] ?

File: packages/knowledge/src/base.ts (hunk 3 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:504]: I agree.

File: packages/knowledge/src/base.ts (hunk 4 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:557]: I also confused with Diagram and Graph.
In https://datavizproject.com/#, diagram includes arc diagram... However, in CKB, arc diagram is Graph.

File: packages/knowledge/src/base.ts (hunk 1 above)
Comment (cxxxxxn) [reply packages/knowledge/src/base.ts:453]: emmm

File: packages/knowledge/src/base.ts (hunk 4 above)
Comment (neoddish) [reply packages/knowledge/src/base.ts:557]: maybe data-cat / info-cat

**Comment by neoddish** [comment]
Let's keep this until the refactoring of CKB...

**Pull request closed without merging**